conf/crg.config

```diff
@@ -5,11 +5,30 @@ params {
     config_profile_url = 'http://www.linux.crg.es/index.php/Main_Page'
 }
 
-process {
-    executor = 'crg'
-    queue    = 'short-centos79,long-centos79'
+def host = System.getenv("HOSTNAME")
+
+if ( host == "genoa64" ) {
```

Comment:
Hey @JoseEspinosa, you cannot use `==` here as the host names are something like `genoa64-03`. You need to use [pattern matching](https://www.baeldung.com/groovy-pattern-matching)